Skyra.Gateway/Program.cs

```diff
@@ -10,20 +10,19 @@ public static class Program
 	{
 		public static async Task Main()
```

Comment:
Why are changes to this in this PR

Reply:
Because I got bored of waiting and because I pushed to the wrong branch